sechub-wrapper-xray/src/main/java/com/mercedesbenz/sechub/wrapper/xray/api/XrayAPIArtifactoryClient.java

```diff
@@ -0,0 +1,116 @@
+package com.mercedesbenz.sechub.wrapper.xray.api;
+
+import java.net.HttpURLConnection;
+import java.util.Objects;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.mercedesbenz.sechub.wrapper.xray.XrayWrapperException;
+import com.mercedesbenz.sechub.wrapper.xray.XrayWrapperJSONConverter;
+import com.mercedesbenz.sechub.wrapper.xray.cli.XrayWrapperArtifactoryClientSupport;
+import com.mercedesbenz.sechub.wrapper.xray.config.XrayWrapperArtifact;
+import com.mercedesbenz.sechub.wrapper.xray.config.XrayWrapperConfiguration;
+
+public class XrayAPIArtifactoryClient {
+
+    private static final Logger LOG = LoggerFactory.getLogger(XrayAPIArtifactoryClient.class);
+
+    private final XrayWrapperArtifact artifact;
+
+    private final XrayWrapperConfiguration xrayWrapperConfiguration;
+
+    XrayAPIHTTPUrlConnectionFactory apiHttpUrlConnectionFactory = new XrayAPIHTTPUrlConnectionFactory();
+
+    XrayAPIResponseFactory xrayAPIResponseFactory = new XrayAPIResponseFactory();
+
+    public XrayAPIArtifactoryClient(XrayWrapperArtifact artifact, XrayWrapperConfiguration xrayWrapperConfiguration) {
+        this.artifact = artifact;
+        this.xrayWrapperConfiguration = xrayWrapperConfiguration;
+    }
+
+    public String requestXrayVersion() throws XrayWrapperException {
+        XrayAPIRequest request = XrayAPIRequestFactory.createGetXrayVersionRequest(xrayWrapperConfiguration.getArtifactory());
+        XrayAPIResponse response = send(request);
+        assertNoError(response, "Could not get Xray Version from Artifactory");
```

Comment:
Just a suggestion: Sometimes a blank line between code parts make it more readable
```suggestion

        assertNoError(response, "Could not get Xray Version from Artifactory");
        
```